components/awards.js

```diff
@@ -0,0 +1,89 @@
+const SteamCommunity = require('../index.js');
+
+SteamCommunity.TargetType = {
+	"UserReview": 1,
+	"SharedFile": 2,
+	"Profile": 3,
+	"Topic": 4,
+	"Comment": 5
+};
+
+SteamCommunity.prototype.getPointsSummary = function(callback) {
+	this.httpRequestGet({
+		"uri": "https://api.steampowered.com/ILoyaltyRewardsService/GetSummary/v1?access_token=" +
+			this.getAccessToken() + "&input_json=%7B%22steamid%22:%22" +
+			this.steamID.toString() + "%22%7D",
```

Comment:
Why are you manually banging out json lol

Reply:
Easier/shorter, but maybe less clean. If I wanted to do it fully proper, I should use `qs`. I have no problem doing that, but I was just trying to be consistent, as you seem to do it this way in the rest of the codebase.

Reply:
Definitely less clean. Please use either JSON.stringify or qs.